scripts/deblur

```diff
@@ -409,40 +411,43 @@ def build_biom_table(seqs_fp, output_biom_fp, min_reads, file_type, log_level,
               type=click.Path(resolve_path=True, readable=True, exists=False,
                               dir_okay=True),
               help="Directory path to store output including BIOM table")
+@click.option('--trim-length', '-t', required=True, type=int,
+              show_default=True, help="Sequence trim length")
 @click.option('--pos-ref-fp', required=False, multiple=True,
               default=[], show_default=False,
               type=click.Path(resolve_path=True, readable=True, exists=True,
                               file_okay=True),
               help="Positive (16S) filtering database. "
                    "Keep all sequences aligning to this FASTA file "
-                   "(for multiple databases, use "
-                   "--ref-fp db1.fa --ref-fp db2.fa ..)\n"
+                   "in the output final.only16s.biom file. (for multiple "
```

Comment:
changed to:
all.biom - all deblurred sequences
positive-match.biom - only sequences matching positive database (16S)
positive-nonmatch.biom - only the non 16S sequences